src/tests/constants.spec.js

```diff
@@ -0,0 +1,49 @@
+import chai from 'chai';
+import {days, defaultProduct, money, users} from './constants';
+
+const {assert, expect} = chai;
+chai.should();
+
+describe('days', () => {
+  it('days should be instance of Array', () => {
+    days.should.be.an.instanceOf(Array);
```

Comment:
this test is not necessary. We test functions, classes, methods, DOM manipulation. Do not need to check array is array